### ISSUE TITLE
Disable docblock comment errors

### ISSUE THREAD

**Issue by nixoncode** [junstyle/vscode-php-cs-fixer#107]
How to disable errors for docblocks only?

**Comment by junstyle** [comment]
error? more info pls

**Comment by nixoncode** [comment]
Sorry! First, I have everything setup as required
- Running PHP 7.4.3

I have an index file with basically nothing except for an echo

```php
<?php

echo 'Hello World';
```

I have a red squiggly on line 2 with the following error message **Missing file doc commentphpcs**

When I add the doc block it show more error, see below
![image](https://user-images.githubusercontent.com/15726312/75631416-c059f680-5c03-11ea-8ff5-29b0181d3552.png)


Basically, I want to continue using it to fix other useful problems but only disable the need to add the docblocs and the tags required on every file/function or class

**Comment by nixoncode** [comment]
I know it needs me to add the tags but that would be too much for me. I just want to suppress these errors  or have them displayed as weak warnings

**Comment by junstyle** [comment]
the error tip produced by **phpcs**, not this extension **php-cs-fixer**, this extension only format php code, do not fix php code

**Comment by nixoncode** [comment]
Thanks, Wrong Repo
